syntax = "proto3";

package command_service;

service CommandService {
  rpc RunCommand (CommandRequest) returns (CommandResponse) {}
}

message CommandRequest {
  string command = 1;
}

message CommandResponse {
  string output = 1;
}